jvm-testing/src/main/kotlin/app/cash/trifle/testing/TestCertificateAuthority.kt

```diff
@@ -32,11 +32,15 @@ data class TestCertificateAuthority(
     )
   }
 
-  fun createTestEndEntity(entityName: String = Random.nextInt().toString()): TestEndEntity {
+  fun createTestEndEntity(entityName: String = Random.nextInt().toString(),
+        validity: Int = -1): TestEndEntity {
```

Comment:
lets set the default value to be the same as `MOBILE_CERTIFICATE_VALIDITY_PERIOD_DAYS`, and just add the argument to the callsite:
```
val certificate = certificateAuthority.signCertificate(rootCertificate, certRequest, validity)
```

Reply:
Chatted and was convinced. However, as a sentinel value (instead of -1), maybe we should change the parameter type to
```
validity: Period? or Int? = null
```
and the branching to be
```
   val certificate: Certificate = if (validity == null) {
      certificateAuthority.signCertificate(rootCertificate, certRequest)
   } else {
      certificate = certificateAuthority.signCertificate(rootCertificate, certRequest, validity)
   }
```